### DISCUSSION TITLE
Reading Users from MDM in QField

### DISCUSSION THREAD

**Discussion by mahmoodi-source** [opengisch/QField#5768] [Q&A]
Hi,
I’m interested in knowing whether QField supports the ability to read user information from Mobile Device Management (MDM) systems. Specifically, can QField extract or utilize user profiles managed through MDM for authentication or access control purposes? Any details on this functionality would be greatly appreciated.
Thank you!

**Comment by m-kuhn** [comment]
This is not supported at the moment and would require development.
